packages/babel-core/src/transformation/file/options/option-manager.js

```diff
@@ -268,6 +273,7 @@ export default class OptionManager {
         val = require(presetLoc);
       }
 
+
```

Comment:
Misunderstood it was just the extra blank line not needed. Pushing up stream again in a moment.